.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,11 @@
-Description:
-<!--
-Replace this template with your PR description.
-Please remember to keep in mind the security levels outlined in
-[CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md) and to
-include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title
-
-* **no**: None
-* **low**: Low chance of potential impact to, or exposure of patient data
-* **moderate**: Moderate chance of potential impact to, or exposure of patient data
-* **severe**: Severe chance of potential impact to, or exposure of patient data
-
-Please also:
-
-* Get thumbs from reviewer(s)
-* Verify all tests go green, including CI tests
```

Comment:
Opinion: All items in the PR checklist should be actionable at the time of creating the PR.

Reply:
I would prefer a reminder for this instead of a checklist item:
> If you decide to merge with any failing checks, add an explanatory comment before doing so.

This says, "We are going to allow you to make this judgment call, but need an explanation both for trust building and compliance."

Reply:
Fair. I'll add a reminder.